tests/archive/xtest_2022_05_24.py

```diff
@@ -5,7 +5,7 @@
 
 from brownie import interface, chain
 
-from scripts.vote_2022_05_24 import start_vote
```

Comment:
This is already pushed to master in other PR as well as moving files to archive.

Reply:
Fixed, thanks